chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/ResourcePermissions.java

```diff
@@ -358,6 +363,11 @@ public boolean hasPermissionOnUser(UserDTO user) {
         } else if (isUserRoleOncStaff()) {
             return getRoleByUserId(user.getId()).getAuthority().equalsIgnoreCase(Authority.ROLE_ONC_STAFF);
         } else if (isUserRoleAcbAdmin()) {
+            if (ff4j.check(FeatureList.ROLE_DEVELOPER)) {
+                if (getRoleByUserId(user.getId()).getAuthority().equalsIgnoreCase(Authority.ROLE_DEVELOPER)) {
+                    return true;
```

Comment:
I think the comment at the top of this method needs an update based on this change or possibly just removed. Can we add any unit tests?